### ISSUE TITLE
Bug: When opening Files, it will close after 1–2 seconds

### ISSUE THREAD

**Issue by wharjack** [files-community/Files#12370]
### Description

When I enable “Toggle the preview pane”, it will automatically close when I close it and open it again
**Screenshot**
![image](https://github.com/files-community/Files/assets/52787179/44b6cd43-095e-476b-815f-a462aa1a2774)

From Windows Event Viewer
```
Category: Files.App.App
EventId: 0

A cycle occurred while laying out the GUI.

Exception: 
Microsoft.UI.Xaml.LayoutCycleException: A cycle occurred while laying out the GUI.
```

**Solution**
I found out that by disabling the “Toggle the preview pane” fixes the problem.

![image](https://github.com/files-community/Files/assets/52787179/cda9c7c4-d5c5-4da8-98be-b39721ed435e)

### Steps To Reproduce

1. Open Files App
2. Click the three dots "..." or "See more" in the upper right corner
3. Click "New Pane"
4. Enable "Toggle the preview pane" or press "Ctrl + P"
5. Close the Files App
6. Open it again

### Requirements

Fix crash when opening Files App because of adding a New Pane in Home and enabling “Toggle the preview pane”

### Files Version

Version: 2.4.71.0, Windows Version: 10.0.19044.2965

### Windows Version

Windows 10 Pro 21H2 27/‎08/‎2022 19044.2965 Windows Feature Experience Pack 1000.19041.1000.0

### Log file

[debug.log](https://github.com/files-community/Files/files/11491638/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report. I am unable to reproduce this, Can you give the following information.
- Can you see if this is an issue in the preview version. https://files.community/download

**Comment by yaira2** [comment]
@Josh65-2201 this type of issue is usually dependent on the screen size and resolution, at the end of the day it's an issue with WinUI but there are workarounds that sometimes help. The main challenge is being able to reproduce the issue to work on a fix.

**Comment by hishitetsu** [comment]
I reproduced by following steps.
1. Enable the Tags widget.
2. Enable the preview pane.
3. Star the dual pane mode.

**Comment by wharjack** [comment]
@hishitetsu thank you so much

![image](https://github.com/files-community/Files/assets/52787179/badf3e0b-daba-4dba-9a54-560f3df56bb5)

I forgot I change some of its settings

**Comment by hishitetsu** [comment]
The crash seems to occur when the home page is too long vertically. The Tags widget tends to be longer and crashes more easily, but this is not a problem with the Tags widget itself.

**Comment by 0x5bfa** [comment]
@hishitetsu Do you think this is related issue in the WinUI team? `https://github.com/microsoft/microsoft-ui-xaml/issues/7768`

**Comment by yaira2** [comment]
> @hishitetsu Do you think this is related issue in the WinUI team? `https://github.com/microsoft/microsoft-ui-xaml/issues/7768`

It's the same issue, that said, the workaround mentioned in that issue won't work for us because we're on WinUI 3.

**Comment by yaira2** [comment]
@wharjack this issue will be resolved in the upcoming v2.5.20 release.
